src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -997,6 +1009,31 @@ void extractUniqueFileExtensions(IBaseDataObject p) {
         }
     }
 
+    /**
+     * Checks the Original-Filename and FILE_ABSOLUTEPATH for the filename of the object. Returns a list with of the
+     * non-empty strings found in these fields. If nothing is found in either field, return an empty list.
+     *
+     * @param d The IBDO
+     * @return The list of filenames found in the field Original-Filename or FILE_ABSOLUTEPATH
+     */
+    public static List<String> getBestFilenames(IBaseDataObject d) {
```

Comment:
Consider renaming to clarify that the values come from IBDO parameters, not from the BDO's filename field.
```suggestion
    public static List<String> getFullFilePathsFromParams(IBaseDataObject d) {
```